kotlin/android/build.gradle.kts

```diff
@@ -0,0 +1,104 @@
+plugins {
+    kotlin("jvm")
+    id("com.google.protobuf")
+    id("maven-publish")
+}
+
+dependencies {
+    // Descriptor.proto is included because there is no lite version of common protos:
+    // https://github.com/protocolbuffers/protobuf/issues/1889
+    protobuf(files("./google-protos"))
+    protobuf(files(fileTree("../../proto") {
+        include("*.proto")
+        // webhook is temporarily excluded because the kotlin code generation creates non-compiling code
+        // in the case of a proto with a package that shares a name with a message field.
```

Comment:
LGTM but once we have a ticket we should prolly link here

Reply:
It looks like we don't need the class that is generating with a syntax error, so there is no issue.